targets/csharp-unityv2/source/Shared/Internal/PlayFabVersion.cs

```diff
@@ -0,0 +1,14 @@
+
+namespace PlayFab.Internal
+{
+    public class PlayFabVersion
+    {
+        public static string SdkRevision = "0.23.160414";
+
+        public static string getVersionString()
```

Comment:
All sdks have been converted to use VersionString, including V1.
change this to a const string, rather than a function.
And ideally move it to PlayFabSettings.